UITests/CheckmarxTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FlaUI.Core.AutomationElements;
+using System.Threading.Tasks;
+
+namespace UITests
+{
+    [TestClass]
+    public class CheckmarxTests : BaseTest
+    {
+        [ClassInitialize]
+        public static void ClassInit(TestContext context)
+        {
+            ClassInitialize(context);
+        }
+
+        [TestMethod]
+        public async Task OpenCheckmarxWindow()
```

Comment:
I assume that we can get some exception during this method
no?
if yes - better to add try-catch to this logic